vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.h

```diff
@@ -566,8 +566,10 @@ class LedgerImpl : public ledger::Ledger {
     ledger::UnblindedTokenList list,
     ledger::ResultCallback callback);
 
-  virtual void DeleteUnblindedTokens(
-      const std::vector<std::string>& id_list,
+  virtual void RedeemUnblindedTokens(
```

Comment:
This name is a bit confusing, since this function doesn't actually redeem the tokens, but rather marks them as redeemed, or spent. I'm having trouble thinking of a short descriptive name though.

- `MarkUnblindedTokensSpent`
- `UpdateSpentUnblindedTokens`
- ?

Reply:
what about `MarkUblindedTokensAsSpent`?

Reply:
A little long, but obvious 👍

Reply:
done